pkgs/development/r-modules/generate-r-packages.R

```diff
@@ -71,44 +74,29 @@ formatPackage <- function(name, version, sha256, depends, imports, linkingTo) {
     depends <- depends[depends %in% knownPackages]
     depends <- lapply(depends, escapeName)
     depends <- paste(depends)
-    depends <- paste(sort(unique(depends)), collapse=" ")
-    paste0("  ", attr, " = derive2 { name=\"", name, "\"; version=\"", version, "\"; sha256=\"", sha256, "\"; depends=[", depends, "]; };")
+    depends <- sort(unique(depends))
+    list(name=unbox(name), version=unbox(version), sha256=unbox(sha256), depends=depends)
```

Comment:
```suggestion
    list(version=unbox(version), sha256=unbox(sha256), depends=depends)
```

Reply:
Yeah that was what I was thinking, but on second thought maybe we should retain it.